openapi_python_client/parser/openapi.py

```diff
@@ -41,7 +42,7 @@ def from_data(
                 operation: Optional[oai.Operation] = getattr(path_data, method)
                 if operation is None:
                     continue
-                tag = utils.snake_case((operation.tags or ["default"])[0])
+                tag = to_valid_python_identifier(value=(operation.tags or ["default"])[0], prefix="tag")
```

Comment:
Would it be too much work to make `"tag"` configurable, like field prefix?

Reply:
I think it'd be pretty easy to add in a future PR if anyone wants it.